ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -1082,7 +1082,7 @@ tests:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-winc
 - as: aws-ipi-private-f14-disasterrecovery
-  cron: 11 11 10,25 * *
+  cron: 11 11 1,7,13,19,25 * *
```

Comment:
The `fN` in test name needs update to make them in sync. Also the cron entries are generated by tools from tests names as it's hard to maintain manually across platforms/versions/architectures/...

Reply:
First update `fN` as you needed. Then use [generate-cron-entry.sh](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh) to generate individual cron entry, or batch update them with [update-cron-entries.py](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/update-cron-entries.py) which calls `generate-cron-entry.sh`

Reply:
@liangxia Tried update-cron-entries.py, it will update the cron fields with rules itself, but it can not meet our expectation, our Mgr want to execute one dr test per day, and we have 6 different dr testnames in 4.15-nightly.yaml, so I plan config 5 days for each dr testname, and 5 * 6 dr test totally in one month, it require that 5 days are in diff date between each dr test name, but update-cron-entries.py's rule will create the 5 days in A testname have overlap with B testname, 

such as rules create cron field list below:

- as: aws-ipi-private-f6-disasterrecovery
  cron: 45 14 6,**12**,18,23,**30** * *
...................
- as: aws-ipi-private-fips-f6-disasterrecovery
  cron: 9 12 5,**12**,18,24,**30** * *
...................

we may find that in date: 12,30, the two testname have overlap, it means there are some day will miss dr test because there are some overlap in other days.

what I design for cron field by manual are like below(there is not overlap for the date between any two dr test names, it can ensure there is only 1 dr test executed per day):

- as: aws-ipi-private-f14-disasterrecovery
  cron: 11 11 1,7,13,19,25 * *
  .................
- as: aws-ipi-private-fips-f14-disasterrecovery
  cron: 11 11 2,8,14,20,26 * *
  
  

cc: @xingxingxia @wsun1